src/utils/ValidateApiKey.ts

```diff
@@ -0,0 +1,17 @@
+import { Configuration, OpenAIApi } from "openai";
```

Comment:
I believe we can verify without `openai`. with axios and catch.

Reply:
can we use axios instead of sse?

Reply:
Oh nope, i mean for verification. sse for chat is fine